src/pages/atomicdex/api/v20/enable_bch_with_tokens/index.mdx

```diff
@@ -2,240 +2,342 @@
+| ---------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| current_block          | integer   | Block height of the coin being activated                                                                                                         |
+| bch_addresses_infos    | object    | A standard [addressInfos](src/pages/atomicdex/api/v20#addressInfos) object. Note: the structure may vary based on the value of the `get_balances` parameter. |
+| slp_addresses_infos    | object    | A standard [addressInfos](src/pages/atomicdex/api/v20#addressInfos) object. Note: the structure may vary based on the value of the `get_balances` parameter. |
+
+
+### 📌 Examples
+
+#### Request with tx_history, cashaddress format, automated utxo merging and get_balances set to true.
 
 <CodeGroup title="" tag="POST" label="enable_bch_with_tokens">
 ```json {{ mm2MethodDecorate : true }}
 {
-  "userpass": "$userpass",
-  "method": "enable_bch_with_tokens",
-  "mmrpc": "2.0",
-  "params": {
-    "ticker": "BCH",
-    "allow_slp_unsafe_conf": false,
-    "bchd_urls": [
-      "https://bchd.imaginary.cash:8335/"
-    ],
-    "mode": {
-      "rpc": "Electrum",
-      "rpc_data": {
-        "servers": [
-          {
-            "url": "electrum1.cipig.net:10055"
-          },
-          {
-            "url": "electrum2.cipig.net:10055"
-          },
-          {
-            "url": "electrum3.cipig.net:10055"
-          },
-          {
-            "url": "electrum1.cipig.net:20055",
-            "protocol": "SSL"
-          },
-          {
-            "url": "electrum2.cipig.net:20055",
-            "protocol": "SSL"
-          },
-          {
-            "url": "electrum3.cipig.net:20055",
-            "protocol": "SSL"
-          }
-        ]
-      }
-    },
-    "tx_history": true,
-    "slp_tokens_requests": [
-      {
-        "ticker": "ASLP-SLP",
-        "required_confirmations": 4
-      }
-    ],
-    "required_confirmations": 5,
-    "requires_notarization": false,
-    "address_format": {
-      "format": "cashaddress",
-      "network": "bitcoincash"
-    },
-    "utxo_merge_params": {
-      "merge_at": 50,
-      "check_every": 10,
-      "max_merge_at_once": 25
-    }
-  }
+	"userpass": "$userpass",
+	"method": "enable_bch_with_tokens",
+	"mmrpc": "2.0",
+	"params": {
+		"ticker": "BCH",
+		"allow_slp_unsafe_conf": false,
+		"bchd_urls": [
+			"https://bchd.imaginary.cash:8335/"
+		],
+		"mode": {
+			"rpc": "Electrum",
+			"rpc_data": {
+				"servers": [{
+						"url": "electrum1.cipig.net:10055"
+					},
+					{
+						"url": "electrum3.cipig.net:20055",
+						"protocol": "SSL"
+					}
+				]
+			}
+		},
+		"tx_history": true,
+		"get_balances": true,
```

Comment:
`"get_balances": true,` can be removed from the request since this is the default behaviour.